### DISCUSSION TITLE
What is the format of indices_tuple in contrastive loss, Tuplemarginal loss etc ?

### DISCUSSION THREAD

**Discussion by Ehan1han** [KevinMusgrave/pytorch-metric-learning#601] [Q&A]
What is the format of indices_tuple in contrastive loss, Tuplemarginal loss etc ?

**Comment by KevinMusgrave** [comment]
For pair-based losses like ContrastiveLoss, MultiSimilarityLoss etc see this post: https://github.com/KevinMusgrave/pytorch-metric-learning/discussions/435#discussioncomment-2268101

For triplet-based losses, the format is `(a, p, n)` where `a` contains the anchor indices, `p` contains the positive indices, and `n` contains the negative indices.
